charts/lagoon-logs-concentrator/templates/metrics-service.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+{{- with .Values.metrics.service.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+{{- end }}
+  name: {{ include "lagoon-logs-concentrator.fullname" . }}-metrics
+  labels:
+    service: metrics
```

Comment:
As discussed, maybe this should be updated to `metrics-only=true` to match the other charts? But I'm not too fussed. Just whatever is most consistent with other servicemonitor selectors.

Reply:
Updated it so that it aligns with the other charts in this repository.